package.json

```diff
@@ -4,11 +4,14 @@
   "description": "lightweight Disquis alternative",
   "license": "MIT",
   "repository": "github:posativ/isso",
-  "dependencies": {},
```

Comment:
This seems unrelated, or do you need newer versions of these for this change?

Reply:
I’m not sure how that change snuck in, I thought I’d reverted that file to the one on master.